care/emr/migrations/0030_facilityorganization_has_children_old_and_more.py

```
@@ -0,0 +1,64 @@
+        if row.has_children == False:
+            row.has_children_old = row.has_children
+            row.save(update_fields=["has_children_old"])
+            parent = row.parent
+            if parent:
+                recursively_copy_and_fix_has_children_value(parent)
+        else:
+            # this is probably a mistake and currently unsupported, so force set value to False
+            row.has_children_old = row.has_children
+            row.has_children = False
+            row.save(update_fields=["has_children", "has_children_old"])
+            print(row.name)
```

Ruff (0.8.2)

`print` found

(T201)